Cargo.toml

```diff
@@ -457,3 +457,11 @@ incremental = false
 debug-assertions = false
 overflow-checks = false
 opt-level = 's'
+
+[profile.optimized]
+inherits = "release"
+codegen-units = 1
+incremental = false
+lto = true
+panic = "abort"
```

Comment:
This can probably be enabled unconditionally for the c-api. We aren't catching panics on the C abi boundary, so they will abort anyway (or be UB until the `c_unwind` rustc feature is fully stabilized).

Reply:
I am getting the following warning

```
Caused by:
  `panic` may not be specified in a `package` profile
```

It seems as though this is not possible based on the [documentation](https://doc.rust-lang.org/cargo/reference/profiles.html#overrides):

> Overrides cannot specify the `panic`, `lto`, or `rpath` settings.

Reply:
I meant setting the `CARGO_PROFILE_<profile name>_PANIC="abort"` env var in `CMakeLists.txt` where `<profile name>` is either `DEV` or `RELEASE` depending on if `--release` was passed or not.

Reply:
Fixed